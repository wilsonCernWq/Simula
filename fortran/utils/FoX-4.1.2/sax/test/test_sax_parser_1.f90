program test_sax_reader

  use m_common_array_str
  use m_sax_parser
  use m_sax_reader
  use m_sax_types

  type(file_buffer_t) :: fb
  type(sax_parser_t) :: fx

  integer :: iostat

  call open_file(fb, file="test_sax_fsm_1.in", iostat=iostat)

  call sax_parser_init(fx, "test_sax_fsm_1.in")

  call sax_parse(fx, fb)

  write(*,'(i0)') fx%xds%xml_version
  write(*,'(a)') str_vs(fx%xds%encoding)
  write(*,'(l1)') fx%xds%standalone

  call sax_parser_destroy(fx)
  call close_file(fb) 

end program test_sax_reader
