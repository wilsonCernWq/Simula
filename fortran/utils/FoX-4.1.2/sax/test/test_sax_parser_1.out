11
us-ascii
T
